### PR TITLE
KRACOEUS-8895:Add CAS as dependency

### PR DESCRIPTION
**Pull request by blackcathacker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,6 +137,7 @@
         <xmlsec.version>1.5.6</xmlsec.version>
         <xstream.version>1.2.2</xstream.version>
         <extra-enforcer-rules.version>1.0-beta-2</extra-enforcer-rules.version>
+        <cas-client-core.version>3.1.10</cas-client-core.version>
```
Comment (leoherbie) [inline pom.xml]: The latest client is 3.2.1 as reverenced here: https://www.apereo.org/content/jasig-cas-client-java-version-321-releasedI'll try to see why I cannot find it in maven central.

Comment (leoherbie) [inline pom.xml]: Doh.  it actually looks like they are up to version 3.3.3 but changed maven coordinates.  Gah!

File: pom.xml [hunk 2]
```diff
@@ -2019,6 +2019,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+            	<groupId>org.jasig.cas</groupId>
```
Comment (leoherbie) [inline pom.xml]: should be org.jasig.cas.client:cas-client-core:3.3.3

File: pom.xml [hunk 3]
```diff
@@ -332,7 +333,6 @@
                                             <exclude>org.opensymphony.quartz</exclude>
                                             <exclude>opensymphony:quartz</exclude>
                                             <exclude>javassist</exclude>
-                                            <exclude>org.jasig.cas</exclude>
```
Comment (leoherbie) [inline pom.xml]: leave this in since they changed group id

File: coeus-webapp/pom.xml [hunk 4]
```diff
@@ -84,6 +84,17 @@
             <artifactId>coeus-impl</artifactId>
             <version>${project.version}</version>
         </dependency>
+      	<dependency>
+	    	<groupId>org.jasig.cas</groupId>
+	    	<artifactId>cas-client-core</artifactId>
+	    	<scope>runtime</scope>
+	    	<exclusions>
+	    		<exclusion>
```
Comment (leoherbie) [inline coeus-webapp/pom.xml]: I wonder if you still need to exclude this with version 3.3.3?  Also, I wonder if there are any new dependencies being brought in we should consider with version 3.3.3

**Comment by blackcathacker** [comment]
Updated commit. No longer needed the exclusion as it doesn't pull in commons-logging anymore.

File: pom.xml [hunk 5]
```diff
@@ -2019,6 +2020,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+            	<groupId>org.jasig.cas.client</groupId>
+            	<artifactId>cas-client-core</artifactId>
+            	<version>${cas-client-core.version}</version>
```
Comment (leoherbie) [inline pom.xml]: put <scope>runtime</scope> here

File: coeus-webapp/pom.xml [hunk 6]
```diff
@@ -84,6 +84,11 @@
             <artifactId>coeus-impl</artifactId>
             <version>${project.version}</version>
         </dependency>
+      	<dependency>
+	    	<groupId>org.jasig.cas.client</groupId>
+	    	<artifactId>cas-client-core</artifactId>
+	    	<scope>runtime</scope>
```
Comment (leoherbie) [inline coeus-webapp/pom.xml]: remove ```<scope>runtime</scope>``` here

File: pom.xml (hunk 5 above)
Comment (blackcathacker) [inline pom.xml]: Moved scope definition.

**Pull request merged**